### PR TITLE
Add WM03 Solution

### PR DESCRIPTION
**Pull request by jollypolly123**

### CODE REVIEW

File: Web/WM03/README.md [hunk 1]
```diff
@@ -5,7 +5,45 @@
 > Visit the site at https://cfta-wm03.allyourbases.co and find a way to bypass the password check.
 
 ## Solution
+Going to the source page, we can see that the logic for the login is as below:
+```
```
Comment (JoshHeng) [inline Web/WM03/README.md:9]: Use ```html for syntax highlighting

File: Web/WM03/README.md [hunk 2]
```diff
@@ -5,7 +5,45 @@
...
+            }
+            return "Incorrect";
+        };
+    -->
+```
 
+We can exploit the comparison ```==```. Essentially, we can find another hash starting with `0e` that will return true if compared using the ```==``` operator to the given hash. There is more information about this here: https://www.whitehatsec.com/blog/magic-hashes/.
+
+Looking at the above logic, we can see that we need a password that when hashed using MD5 and the given salt, returns a hash that begins with `0e` and is 32 digits long, including the `0e`. Hashes cannot be reversed, so we can instead write a python script that will test hashes until we get the correct one (it takes a while so you might want to run it in the background):
+```
```
Comment (JoshHeng) [inline Web/WM03/README.md:29]: Use ```python for syntax highlighting

**Review by JoshHeng — CHANGES_REQUESTED** [review]
Looks good and thanks for the PR, please just add the language used in codeblocks so it'll highlight syntax

File: Web/WM03/README.md (hunk 1 above)
Comment (jollypolly123) [reply Web/WM03/README.md:9]: got it, thanks!

**Comment by 404dcd** [comment]
LGTM

**Review by 404dcd — APPROVED** [review]

File: Web/WM03/README.md (hunk 1 above)
Comment (RealJammy) [reply Web/WM03/README.md]: fixed in https://github.com/Alic3C/Cyber-FastTrack-Spring-2021/pull/4/commits/2d29456bc21f106fda13f6e6fe940a7b6804a33f

**Review by RealJammy — APPROVED** [review]

**Pull request merged**